### PR TITLE
Cherry pick GDT and GDTCCT fixes into release-6.10.0

### PR DESCRIPTION
**Pull request by mikehaney24**
This PR is for an out of band release.

GDT: 3.0.0 -> 3.0.1
GDTCCT: 1.2.0 -> 1.2.1

### CODE REVIEW

**Comment by mikehaney24** [comment]
> After travis green, please remove the .travis.yml temp change and update the podspec's with the right versions.
> 
> Then, I'll merge and publish

Travis hadn't gotten around to really running the tests, so I updated the podspecs with the correct versions.

**Comment by mikehaney24** [comment]
Travis is green, submitting a change to remove the branch from travis.

image

File: .travis.yml [hunk 1]
```diff
@@ -543,3 +543,4 @@ jobs:
 branches:
   only:
     - master
+    - release-6.10.0
```
Comment (paulb777) [inline .travis.yml]: Actually this is fine to leave in this branch, as long as we delete on the merge back to master.

Comment (mikehaney24) [reply .travis.yml]: Welp. The change is after this.

**Pull request merged**